### PR TITLE
Update Google Analytics plugin to use up to date API

### PR DESCRIPTION
**Pull request by willdunklin**
Removed old google analytics (analytics.js) API which has been [deprecated](https://support.google.com/analytics/answer/11583528).

### CODE REVIEW

File: plugins/google_analytics/girder_google_analytics/web_client/views/ConfigView.js [hunk 1]
```diff
@@ -56,6 +56,8 @@ var ConfigView = View.extend({
             },
             error: null
         }).done(() => {
+            location.reload();
```
Comment (zachmullen) [inline plugins/google_analytics/girder_google_analytics/web_client/views/ConfigView.js:59]: What's this for?

Comment (willdunklin) [reply plugins/google_analytics/girder_google_analytics/web_client/views/ConfigView.js:59]: This clears the previous js bundle from google. If we change the tag we're tracking, we need to request a fresh js bundle from `main.js:initGoogleAnalytics`

Comment (willdunklin) [reply plugins/google_analytics/girder_google_analytics/web_client/views/ConfigView.js:59]: @zachmullen is this an issue?

Comment (zachmullen) [reply plugins/google_analytics/girder_google_analytics/web_client/views/ConfigView.js:59]: I'd say it's generally an anti-pattern to do a hard refresh in a single-page app. Is it not sufficient to re-invoke `initGoogleAnalytics`? Or would the presence of the previous state corrupt the page in some way?

**Review by zachmullen — APPROVED** [review]

File: plugins/google_analytics/girder_google_analytics/web_client/views/ConfigView.js (hunk 1 above)
Comment (willdunklin) [reply plugins/google_analytics/girder_google_analytics/web_client/views/ConfigView.js]: I was initially worried about the prospect of having multiple competing results from `initGoogleAnalytics`'s `$.getScript`, but after testing it further seems like it won't be an issue for usage.

**Comment by willdunklin** [comment]
@zachmullen seems like the CI errors here are outside the scope of this PR's changes (unless I'm mistaken). Are you able to merge this? We have a customer waiting on these changes

**Pull request merged**